ReactAndroid/src/main/java/com/facebook/react/ReactActivityDelegate.java

```diff
@@ -29,6 +30,10 @@
  * class doesn't implement {@link ReactApplication}.
  */
 public class ReactActivityDelegate {
+
+  private final int REQUEST_OVERLAY_CODE = 1111;
```

Comment:
@satya164 The number must be between 0 and 65535 I believe
http://stackoverflow.com/a/37812367/1784299

I don't think Random is the correct approach because then it is "random" for a developer when that clashes. It should be predefined so a developer knows early on why they can't use `1` for example as their request code.

I don't think its that big of a deal to be honest but want to get it "right" the first time.